src/main/resources/customImageTemplateWithManagedDisk.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {},
+  "variables": {
+    "virtualNetworkName": "",
+    "virtualNetworkResourceGroupName": "",
+    "subnetName": "",
+    "nsgName": "",
+    "storageAccountName": "[concat('jnk',uniqueString(resourceGroup().id))]",
```

Comment:
It  just a tag here, when using custom vhd, we delete empty container by matching this `jnk`